### PR TITLE
upgrade nextjs from 6 to 8

### PR DESCRIPTION
**Pull request by kachen1027**
Note: 
1. injectGlobal API being replaced by the new createGlobalStyle helper component in Styled Component version 4
https://www.styled-components.com/docs/faqs#what-do-i-need-to-do-to-migrate-to-v4

2. Synchronous HOC is no longer supported since version 4.0.0 in Next Redux Saga
https://github.com/bmealhouse/next-redux-saga/blob/master/README.md

### CODE REVIEW

**Comment by kachen1027** [comment]
issue #10

File: templates/saga/package.json [hunk 1]
```diff
@@ -1,43 +1,43 @@
 {
-  "name": "create-next-app",
+  "name": "my-saga",
```
Comment (davidnguyen179) [inline templates/saga/package.json]: I think `create-next-app` is more generic than `my-saga`. What do you think if we revert back to the origin?

Comment (kachen1027) [reply templates/saga/package.json]: Ya sure I agree. I think I miss committed this line.

Comment (davidnguyen179) [reply templates/saga/package.json]: So can u commit again?

Comment (kachen1027) [reply templates/saga/package.json]: done

**Comment by davidnguyen179** [comment]
Ok, the final stage is clean the history by using rebase & squash commits. Currently, I saw 2 commits

```bash
upgrade nextjs from 6 to 8
revert name to "create-next-app"
```

It should be like this:
1. Squash 2 commits into 1
2. Update commit message like below

```bash
11: Upgrade nextjs from 6 to 8
```

3. After rebasing & you push force
```bash
git push -f
```

**Comment by kachen1027** [comment]
Done

**Pull request merged**